src/app/shared/components/search/search.component.scss

```diff
@@ -1,19 +1,57 @@
-.search-bar {
-  position: relative;
+input {
+	outline: none;
+}
+input[type=search] {
+	-webkit-appearance: textfield;
+	-webkit-box-sizing: content-box;
+}
+
+input[type=search] {
+	background: url(https://static.tumblr.com/ftv85bp/MIXmud4tx/search-icon.png) no-repeat 9px center;
+	border: solid 1px #ccc;
```

Comment:
Use transparent border color in default state.
    border: solid 1px transparent;